css/main.css

```diff
@@ -1,3 +1,245 @@
+.products {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+.product_item {
+  width: 100%;
+  margin: 1em auto;
+
+  min-height: 80vh;
+  text-align: center;
+}
+.product_contents {
+  display: inline-flex;
+  flex-direction: column;
+  margin-top: 10px;
+  color: var(--red);
+  padding-top: 1em;
+}
+.product_underline {
+  width: 1em;
+  border: 3px solid var(--red);
+  margin: 1em auto;
+}
+.cacti {
+  background: url("../img/cacti.webp");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.plants {
+  background: url("../img/plants.webp");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+.succulents {
+  background: url("../img/succulents.webp");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+/* =============================About Us========================== */
+
+.aboutus {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(7, 1fr);
+  text-align: center;
+  color: var(--red);
+  line-height: 1.5em;
+  font-size: 5vw;
+  font-weight: bold;
+}
+.aboutus img {
+  width: 100%;
+  max-width: 100%;
+}
+.aboutus_text {
+  grid-row: 1/3;
+  grid-column: 1/-1;
+  padding: 2em;
+  background: var(--white);
+}
+.first_small_section {
+  grid-column: 1/-1;
+  grid-row: 3/4;
+  background: url("../img/first_small.webp");
+  background-size: cover;
+}
+.second_small_section {
+  grid-column: 1/-1;
+  grid-row: 4/5;
+  background: url("../img/second_small.webp");
+  background-size: cover;
+}
+.first_big_section {
+  grid-column: 1/-1;
+  grid-row: 5/6;
+  background: url("../img/bigabout.webp");
+  background-size: cover;
+}
+.second_big_section {
+  grid-column: 1/-1;
+  grid-row: 6/8;
+  background: var(--black);
+  color: var(--white);
+}
+.second_big_section h1 {
+  margin: 2em auto 1em auto;
+  font-size: 14vw;
+  line-height: 1.5em;
+}
+.aboutus_text h1 {
+  font-size: 8vw;
+}
+.about_btn {
+  padding: 1em 1.5em;
+  background: var(--white);
+  color: var(--black);
+  margin-top: 2em;
+}
+/* ==================================Float BTNs============ */
+.float_btn {
+  width: 1em;
+  height: 1em;
+  text-align: center;
+  font-size: 15vw;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  border-radius: 50%;
+  padding: 0;
+  font-weight: bold;
+}
+.float_btn2 {
+  position: fixed;
+  bottom: 0.5em;
+  right: 0.5em;
+  border-radius: 50%;
+  padding: 1em;
+  font-size: 3vw;
+}
+.float_btn2 img {
+  width: 100%;
+  max-width: 100%;
+  display: block;
+  margin: 0 auto;
+}
+@media screen and (min-width: 481px) {
+  .hero {
+    background: url("../img/big_hero.webp");
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+  }
+  .second_hero_section h1 {
+    font-size: 8vw;
+  }
+  .third_hero_section h2 {
+    font-size: 5vw;
+  }
+  .cacti,
+  .plants,
+  .succulents {
+    width: 30%;
+    padding: 1em 0;
+    background-position: center;
+  }
```

Comment:
Why not target the `product_item` class directly here? 

As a suggestion, using grid might be easier here. 3 columns of `1fr` each, a column-gap, and you're done 😉

Reply:
Yes you're right 😃 Thanks!